testsuite/openshift/objects/auth_config/sections.py

```diff
@@ -161,6 +161,9 @@ def uma_metadata(self, name, endpoint, credentials, **common_features):
 class ResponsesSection(Section, Responses):
     """Section which contains response configuration"""
 
+    def add_simple(self, auth_json, name="simple", key="data", **common_features):
```

Comment:
Is it possible to also change response to simple in the `test_response_condition.py` authorization fixture? As simple response also supports conditional common feature?

Reply:
Response module has custom fixture for manipulating responses. The method is mesnt more for tests that use response as a tool instead of a actual thing to test